### DISCUSSION TITLE
How can I change the media storage path to another folder on another SSD drive

### DISCUSSION THREAD

**Discussion by Yurch24** [mediacms-io/mediacms#1000] [General]
The SSD drive where the media storage folder is located by default is too small, there was a need to redirect the storage to another SSD drive. But it is not possible to implement this idea. I will be grateful for your help.

**Comment by AnthonyTorreso** [comment]
Not 100% sure, but I think this is what your looking for.
edit:   /home/mediacms.io/mediacms/cms/settings.py

I think this is where you edit your dir's

```
TEMP_DIRECTORY = "/tmp"  # Don't use a temp directory inside BASE_DIR!!!
BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
STATIC_URL = "/static/"  # where js/css files are stored on the filesystem
MEDIA_URL = "/media/"  # URL where static files are served from the server
STATIC_ROOT = BASE_DIR + "/static/"
# where uploaded + encoded media are stored
MEDIA_ROOT = BASE_DIR + "/media_files/"                    <<--------

```
or you may just want to mount your secondary drive as the /home/mediacms.io/mediacms/media_files2 and then edit that MEDIA_ROOT however I don't know if it will then break the already uploaded files.    Heck give it a try and let us all know.

Thanks,
Anthony

**Comment by jackson-xp** [comment]
Why not use logical volume management (LVM) to expand the original disk? This is the best approach. Of course, there's also a clever way: copying the old data files to the new larger disk, modifying all permissions and attributes to match the original, then deleting the files on the old disk and creating a symbolic link pointing to the new data disk files. This method is also very reliable.Doing so means no configuration changes are made to the system.
